kinder/pkg/cluster/manager/actions/kubeadm-config.go

```diff
@@ -92,8 +92,8 @@ func KubeadmConfig(c *status.Cluster, kubeDNS bool, copyCertsMode CopyCertsMode,
 		APIBindPort:          constants.APIServerPort,
 		APIServerAddress:     controlPlaneIP,
 		Token:                constants.Token,
-		PodSubnet:            "192.168.0.0/16", // default for calico
-		ServiceSubnet:        "",               // let kubeadm apply default
+		PodSubnet:            "192.168.0.0/16",
```

Comment:
oddly this failed for me locally...or maybe i'm mistaken because the CI test passed on this PR.

Reply:
It looks like `PodSubnet` is also required by `kindnet`.
https://github.com/SataQiu/kindnet/blob/d3c882ff20054f504e3bb7cf3637c74cf9a332e4/cmd/kindnetd/subnets.go#L65-L72